assets/prefabs/food/raw/Starfruit.prehab

```diff
@@ -0,0 +1,26 @@
+{
+    "parent": "engine:iconItem",
+    "Item": {
+        "icon": "Cooking: starfruit”,
```

Comment:
The second last character in this line is a stray inverted comma. Please fix this :)